README.md

```diff
@@ -90,15 +92,6 @@ Versions are auto incremented based on commit messages and SemVer (Major.Minor.P
 - `patch:` or `fix:` example `fix: url bug` 
   - this will increment the PATCH version eg 1.0.0 to 1.0.1. This will also happen by default.
 
-#### Build process
-
-1. Pushes to any branch will start the build process
```

Comment:
Hi! Please change to:

1. When you push to any branch, it will start the build process.

Reply:
these are line removals, please ignore!